app/models/book.rb

```diff
@@ -146,6 +884,71 @@ def create_teacher_set_version_on_update
   end
 
   def update_from_isbn
-    # later we will call the BibService to get more fields for this new book
+    # response = send_request_to_bibs_microservice(isbn)
+    response = EXAMPLE_RESPONSE_BODY_FROM_BIB_SERVICE
```

Comment:
Blocker: `send_request_to_bibs_microservice(isbn)` is receiving a 401 so we're mocking the response for now.

Navraj helped me look into it this morning.  I'll keep looking.